src/app/content/highlights/components/ColorPicker.tsx

```diff
@@ -144,15 +144,24 @@ const ColorPicker = ({className, ...props}: Props) => {
             ? props.onRemove ? props.onRemove() : null
             : props.onChange(style.label)}
       />)}
-    </div>
+    </fieldset>
   );
 };
 
 export default styled(ColorPicker)`
+  border: 0;
   outline: none;
   display: flex;
   flex-direction: row;
   overflow: visible;
   gap: ${cardPadding}rem;
   padding: ${cardPadding}rem 0.3rem;
+
+  legend {
+    position: absolute;
+    height: 1px;
+    width: 1px;
+    overflow: hidden;
+    clip: rect(1px, 1px, 1px, 1px);
```

Comment:
I could see this becoming a reusable const if you think it's generic enough for most contexts.

Reply:
I think it is defined as a constant in one of the other PRs.